docs/workflows/alternate-content-source.rst

```diff
@@ -0,0 +1,54 @@
+Alternate Content Sources
+=========================
+
+Alternate Content Sources can help speed up populating of new repositories.
+If you have content stored locally or geographically near you, it allows
+you its' faster transfer. Alternate Content Sources will allow you to use
+the content you have around if it matches the remote one.
+
+`Alternate Content Sources <https://docs.pulpproject.org/pulpcore/workflows/alternate-content-sources.html>`_
+base is provided by pulpcore plugin.
+
+To use an Alternate Content Source you need a ``RPMRemote`` with path of your ACS.
+
+.. warning::
+
+    Mirror list cannot be used as an Alternative Content Source.
+
+.. code-block:: bash
+
+    http POST $BASE_ADDR/pulp/api/v3/remotes/rpm/rpm/ name="myRemoteAcs" policy="on_demand" url="http://fixtures.pulpproject.org/rpm-unsigned/"
+
+Create Alternate Content Source
+-------------------------------
```

Comment:
Somewhere in here, write `Alternate Content Source (ACS)` to explicitly define the acronym before using it.